applications/AdjointFluidApplication/CMakeLists.txt

```diff
@@ -2,7 +2,14 @@ set(CMAKE_INCLUDE_CURRENT_DIR ON)
 
 message("**** configuring KratosAdjointFluidApplication ****")
 
-include_directories( ${CMAKE_SOURCE_DIR}/kratos )
+if( NOT DEFINED EIGEN_ROOT )
+	message( FATAL_ERROR "EIGEN_ROOT not defined" )
```

Comment:
If you do this you cannot compile AdjointFluid without Eigen!
This has to be optional

Reply:
made the eigen dependency optional